dev-docs/source/UsefulTools.rst

```diff
@@ -0,0 +1,156 @@
+Useful Tools for Developers
+===========================
+
+Creating classes: class_maker.py
+--------------------------------
+
+**To make it faster to create a new class**. This is a small python
+script located in /buildconfig/. It generates the .cpp, .h and test
+files for a class along with some code stubs. It can also flesh out more
+methods for new Algorithms, using the "--alg" option.
+
+| ``usage: class_maker.py [-h] [--force] [--test] [--alg] SUBPROJECT CLASSNAME``
```

Comment:
The back ticks on each line make this quite hard to read, how about a [literal block](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks)?